### PR TITLE
Column order fixes

### PR DESCRIPTION
**Pull request by klumba12**

### CODE REVIEW

File: src/core/navigation/navigation.js [hunk 1]
```diff
@@ -17,157 +15,230 @@ export default class Navigation {
...
-					if (navigationState.row == -1 && navigationState.column == -1) {
-						model.navigation({
-							column: 0,
-							row: 0
-						});
-					} else {
-						model.navigation({row: navigationState.row + 1});
-					}
-				}
+				canExecute: () => canExecute() && this.nextRow >= 0,
```
Comment (etokareva) [inline src/core/navigation/navigation.js:154]: Why we use canExecute inside of canExecute?

Comment (klumba12) [reply src/core/navigation/navigation.js:154]: this is common canExecute logic for all navigation commands, What name do you prefer?

Comment (etokareva) [reply src/core/navigation/navigation.js:154]: Just don't fully understand how the code works

Comment (klumba12) [reply src/core/navigation/navigation.js:154]: just look on canExecute method that is defined above

**Review by titovmx — APPROVED** [review]

**Pull request merged**